data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -0,0 +1,498 @@
+                        date_column=date_row.date_column)
+                    queries_list.append({
+                        clean_consts.QUERY: clean_query,
+                        clean_consts.DESTINATION_DATASET: date_row.dataset_id,
+                        clean_consts.DESTINATION_TABLE: date_row.table,
+                        clean_consts.DISPOSITION: bq_consts.WRITE_TRUNCATE,
+                        'type': 'retraction'
+                    })
+                else:
+                    sandbox_query = SANDBOX_QUERY_END_DATE.render(
+                        project=date_row.project_id,
+                        sandbox_dataset=sandbox_dataset,
+                        intermediary_table=ticket_number + '_' + date_row.table,
+                        dataset=date_row.dataset_id,
+                        table=date_row.table,
+                        pid=pid,
+                        deactivated_pids_project=pids_project_id,
+                        deactivated_pids_dataset=pids_dataset_id,
+                        deactivated_pids_table=pids_table,
+                        end_date_column=date_row.end_date_column,
+                        start_date_column=date_row.start_date_column)
+                    queries_list.append({
+                        clean_consts.QUERY: sandbox_query,
+                        clean_consts.DESTINATION_DATASET: sandbox_dataset,
+                        clean_consts.DESTINATION_TABLE: date_row.table,
+                        'type': 'sandbox'
+                    })
+                    clean_query = CLEAN_QUERY_END_DATE.render(
+                        project=date_row.project_id,
+                        dataset=date_row.dataset_id,
+                        table=date_row.table,
+                        pid=pid,
+                        deactivated_pids_project=pids_project_id,
+                        deactivated_pids_dataset=pids_dataset_id,
+                        deactivated_pids_table=pids_table,
+                        end_date_column=date_row.end_date_column,
+                        start_date_column=date_row.start_date_column)
+                    queries_list.append({
+                        clean_consts.QUERY: clean_query,
+                        clean_consts.DESTINATION_DATASET: date_row.dataset_id,
+                        clean_consts.DESTINATION_TABLE: date_row.table,
+                        clean_consts.DISPOSITION: bq_consts.WRITE_TRUNCATE,
+                        'type': 'retraction'
+                    })
+            else:
+                # break out of loop to create query, if pid does not exist in table
+                continue
+    LOGGER.info(
+        f"Query list complete, retracting ehr deactivated PIDS from the following datasets: "
+        f"{dataset_list}")
+    return queries_list
+
+
+def run_queries(queries, client):
+    """
+    Function that will perform the retraction.
+
+    :param queries: list of queries to run retraction
+    """
+    query_job_ids = []
```

Comment:
Is this still required?  It's set, but I don't think it's used anymore.?.

Reply:
100% this can be removed. I was copying from nishanth's retraction script but realize this is not needed for mine.